lib/texter.rb

```diff
@@ -0,0 +1,26 @@
+require_relative 'takeaway'
+
+require 'twilio-ruby'
+
+class Texter
+
+
+
+  def initialize
+    @client = Twilio::REST::Client.new ENV['ACCOUNT_SID'], ENV['AUTH_TOKEN']
+  end
+  
+  def send
+    'message sent'
+    @client.account.messages.create({
+    :from => '441227641157', 
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Use 2 spaces for indentation in a hash, relative to the first position after the preceding left parenthesis.<br>Trailing whitespace detected.